tests/test_metrics.py

```diff
@@ -27,25 +27,50 @@ def test_compute_not_implemented(
         with pytest.raises(NotImplementedError):
             metric.compute(mock_forecast_dataarray, mock_gridded_obs_dataarray)
 
-    def test_align_datasets(self, mock_forecast_dataarray, mock_gridded_obs_dataarray):
-        """Test that the conversion from init time to valid time (named as time) produces an aligned
-        dataarray to ensure metrics are applied properly."""
+    """Tests the _align_observations_temporal_resolution method of the Metric base class."""
+
```

Comment:
You don't need this doc string, the examples are self-explanatory. If it were very important to split out the tests for this functionality, you'd make a new `class` that only tests these functions, e.g. `class TestMetricObsAlignment` or something.